catchup/catchpointService.go

```diff
@@ -471,10 +471,12 @@ func (cs *CatchpointCatchupService) processStageBlocksDownload() (err error) {
 		return cs.abort(fmt.Errorf("processStageBlocksDownload failed, unable to ensure first block : %v", err))
 	}
 
-	// pick the lookback with the greater of either MaxTxnLife or MaxBalLookback
-	lookback := config.Consensus[topBlock.CurrentProtocol].MaxTxnLife
-	if lookback < config.Consensus[topBlock.CurrentProtocol].MaxBalLookback {
-		lookback = config.Consensus[topBlock.CurrentProtocol].MaxBalLookback
+	// pick the lookback with the greater of either (MaxTxnLife+DeeperBlockHeaderHistory)
+	// or MaxBalLookback
+	proto := config.Consensus[topBlock.CurrentProtocol]
+	lookback := proto.MaxTxnLife + proto.DeeperBlockHeaderHistory
```

Comment:
**Suggestion**: there are 4 places in the code where we simply add these two together - while simple math, it's preferable this logic is centralized within a function for maintainability purposes (especially if this formula needs to change down the line). 

Options could be to a) create a lookback struct in the data/basics package (holding these two fields) with a function that outputs a value based off the underlying simple math, or b) add a function to consensusparams yielding the lookback value (feels dirty)